src/summary/profile/demographics-card.test.tsx

```diff
@@ -0,0 +1,204 @@
+          state: "State0351tested",
+          postalCode: "60351",
+          country: "Country0351"
+        }
+      ]
+    };
+
+    match = { params: {}, isExact: false, path: "/", url: "/" };
+
+    getPatientMock.mockReset();
+  });
+
+  it("renders the correct age for a 55 year old", () => {
+    patient.birthDate = createBirthdayYearsAgo(55);
+    getPatientMock.mockReturnValue(of(patient));
+    const { getByTitle } = render(
+      <BrowserRouter>
+        <DemographicsCard match={match} />
+      </BrowserRouter>
+    );
+    expect(getByTitle("Age").textContent).toBe("55yr");
+  });
+
+  it("renders the correct age for a 55 year old who had their birthday recently", () => {
+    patient.birthDate = createBirthdayYearsAgo(55, 4);
+    getPatientMock.mockReturnValue(of(patient));
+    const { getByTitle } = render(
+      <BrowserRouter>
+        <DemographicsCard match={match} />
+      </BrowserRouter>
+    );
+    expect(getByTitle("Age").textContent).toBe("55yr");
+  });
+
+  it("renders the correct age for a 18 year old", () => {
+    patient.birthDate = createBirthdayYearsAgo(18);
+    getPatientMock.mockReturnValue(of(patient));
+    const { getByTitle } = render(
+      <BrowserRouter>
+        <DemographicsCard match={match} />
+      </BrowserRouter>
+    );
+    expect(getByTitle("Age").textContent).toBe("18yr");
+  });
+
+  it("renders the correct age for a 17 year old who had their birthday three months ago", () => {
+    patient.birthDate = createBirthdayYearsAgo(17, 3);
+    getPatientMock.mockReturnValue(of(patient));
+    const { getByTitle } = render(
+      <BrowserRouter>
+        <DemographicsCard match={match} />
+      </BrowserRouter>
+    );
+    expect(getByTitle("Age").textContent).toBe("17yr 3mo");
+  });
+
+  it("renders the correct age for a 17 year old who had their birthday three months and one day ago", () => {
+    patient.birthDate = createBirthdayYearsAgo(17, 3, 1);
+    getPatientMock.mockReturnValue(of(patient));
+    const { getByTitle } = render(
+      <BrowserRouter>
+        <DemographicsCard match={match} />
+      </BrowserRouter>
+    );
+    expect(getByTitle("Age").textContent).toBe("17yr 3mo");
+  });
+
+  it("renders the correct age for a 17 year old who had their birthday almost three months ago", () => {
+    patient.birthDate = createBirthdayYearsAgo(17, 2, 27);
+    getPatientMock.mockReturnValue(of(patient));
+    const { getByTitle } = render(
+      <BrowserRouter>
+        <DemographicsCard match={match} />
+      </BrowserRouter>
+    );
+    expect(getByTitle("Age").textContent).toBe("17yr 2mo");
+  });
+
+  it("renders the correct age for a 16 week old baby", () => {
+    patient.birthDate = createBirthdayWeeksAgo(16);
+    getPatientMock.mockReturnValue(of(patient));
+    const { getByTitle } = render(
+      <BrowserRouter>
+        <DemographicsCard match={match} />
+      </BrowserRouter>
+    );
+    expect(getByTitle("Age").textContent).toBe("16w");
```

Comment:
Per http://www.gregoryschmidt.ca/writing/patient-age-display-ehr-conventions , this should be "16 week". Which does seem more readable / rapidly differentiable from other units.

Reply:
Will change